### ISSUE TITLE
Archive shows [Sandbox-mailoutput] messages in search results

### ISSUE THREAD

**Issue by rpcross** [ietf-tools/mailarch#3481]
From mahoney@nostrum.com
Should the following search URL (generated by selecting "Search lists > IETF Mail Archive" from a document's status page):

https://mailarchive.ietf.org/arch/search/?q="<I-D_name>"

Display posts to [Sandbox-mailoutput] by default?

For instance, "https://mailarchive.ietf.org/arch/search/?q=%22draft-ietf-lsr-yang-isis-reverse-metric%22" displays 34 posts to [Sandbox-mailoutput]. As a general reader of an I-D, I expected the list of posts to contain messages discussing the document rather than showing dt debugging messages. A reader unfamiliar with this output would be confused by the messages listed.

**Comment by rpcross** [comment]
From rjsparks@nostrum.com
This suggests that having the sandbox mail output go into mailarchive may not have been the right choice.

We could change the search the datatracker generates to leave the sandbox list out of it, but then we would have other test lists that we might also want to elide.

Maybe lists need configuration at mailarchive to indicate if they should show up in global searches by default or only if explicitly searched...

**Comment by rjsparks** [comment]
I agree that having the sandbox dump its mail into the mailarchive was not the right thing to have done, and we should undo that. Before we do, we should create an alternate way to see what the sandbox would have sent. We need a cleaner way to see what mail would have been sent in a development container, and when that exists, we could reuse it for the online sandbox (which should be run as a development container _anyhow_).

But, I think it is likely that we will have other lists that we _should_ be able to hide from global searches (the test list comes to mind as an existence proof.
